src/media/js/addon/actions/review.js

```diff
@@ -43,42 +55,68 @@ export function fetch() {
 }
 
 
-export function publish(slug) {
+export function publish(addonSlug, versionId) {
+  /* Publish version of an add-on. */
   return (dispatch, getState) => {
     const apiArgs = getState().apiArgs || {};
     const publishUrl = Url(
-      urlJoin(process.env.MKT_API_ROOT, `extensions/queue/${slug}/publish/`)
+      urlJoin(
+        process.env.MKT_API_ROOT,
+        `extensions/extension/${addonSlug}/versions/${versionId}/publish/`
+      )
     ).q(apiArgs);
 
-    dispatch(publishPending(slug));
+    dispatch(publishPending({
+      addonSlug,
```

Comment:
[TIL](https://github.com/lukehoban/es6features#enhanced-object-literals). This is really cool.